src/Nox.Solution/NoxSolution.cs

```diff
@@ -39,6 +39,53 @@ internal void Validate()
         return fullRelationshipModels;
     }
 
+    public List<EntityRelationshipWithType> GetOwnedRelationshipsToCreate(
+        Func<string, Type?> getTypeByNameFunc,
+        Entity entity)
+    {
+        var fullRelationshipModels = new List<EntityRelationshipWithType>();
+
+        if (entity?.OwnedRelationships != null)
+        {
+            foreach (var relationship in entity.OwnedRelationships)
+            {
+                fullRelationshipModels.Add(new EntityRelationshipWithType
+                {
+                    Relationship = relationship,
+                    RelationshipEntityType = getTypeByNameFunc(relationship.Entity)!
+                });
+            }
+        }
+
+        return fullRelationshipModels;
+    }
+
+    public bool IsOwnedEntity(Entity entity)
```

Comment:
this method should be internal, the entity has a property that defines if is owned or not

everyone should use that

Reply:
True, but it causes a different issue. Let's discuss the best way on resolving it